### PR TITLE
Add map & search

### PR DESCRIPTION
**Pull request by jowczar**

### CODE REVIEW

File: frontend/views/Information/Information.js [hunk 1]
```diff
@@ -1,78 +1,255 @@
...
+  'Montenegro',
+  'NorthernIreland',
+  'JanMayen',
+  'Netherlands',
+  'Norway',
+  'Svalbard',
+  'Poland',
+  'Portugal',
+  'Romania',
+  'Russia',
```
Comment (ShanghaiG) [inline frontend/views/Information/Information.js:60]: what is this

File: frontend/views/Information/Information.js [hunk 2]
```diff
@@ -1,78 +1,255 @@
...
+  'Vojvodina',
+  'Slovakia',
+  'Slovenia',
+  'Sweden',
+  'Ukraine',
+  'Vatican',
+  'Wales'
+];
+
+// TODO: this needs a second look after we've got list of countries that AI works with
```
Comment (ShanghaiG) [inline frontend/views/Information/Information.js:73]: is ukraine not working ? its hot topic right now ;]

**Review by ShanghaiG — APPROVED** [review]

File: frontend/views/Information/Information.js (hunk 1 above)
Comment (jowczar) [reply frontend/views/Information/Information.js:60]: 🧨

File: frontend/views/Information/Information.js (hunk 2 above)
Comment (jowczar) [reply frontend/views/Information/Information.js:73]: no AI for ukraine.
no AI for these too but nobody knows

**Pull request merged**